src/devices/Mcp25xxx/ReadStatusResponse.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Mcp25xxx
+{
+    /// <summary>
+    /// Response from READ STATUS instruction.
+    /// </summary>
+    public class ReadStatusResponse
```

Comment:
other way - perhaps simpler to represent this:
- have an enum like i.e.:
```csharp
[Flags]
enum StatusResponse : byte
{
  Rx0If = 1 << 0,
  Rx1If = 1 << 1,
  Tx0Req = 1 << 2,
  // ...
}
```

now you don't need to have the class and can just check flags with HasFlag

Reply:
I like this.  Will update, but just use value of bit instead of using the bit shift.